thinkbox-deadline/azuredeploy.json

```diff
@@ -0,0 +1,404 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+
+    "parameters": {
+
+        "location": 
+        {
+            "type": "string",
+            "allowedValues": 
+            [
+                "West US",
+                "East US",
+                "West Europe",
+                "East Asia",
+                "Southeast Asia"
+            ],
+            "metadata": 
+            {
+                "description": "The region to deploy the resources into"
+            }
+        },
+
+        "publicDnsName": 
+        {
+            "type": "string",
+            "metadata": 
+            {
+                "description": "Unique public DNS prefix for the deployment. The fqdn will look something like '<dnsname>.westus.cloudapp.azure.com'. Up to 62 chars, digits or dashes, lowercase, should start with a letter: must conform to '^[a-z][a-z0-9-]{1,61}[a-z0-9]$'."
+            }
+        },
+
+        "storageAccount": 

```

Comment:
storageAccount is a reserved keyword when deploying through PS cmdlets. Consider renaming this to newStorageAccount or something else